.pre-commit-config.yaml

```diff
@@ -11,6 +11,17 @@ repos:
     -   id: check-yaml
     #-   id: check-json
     -   id: check-added-large-files
+        args: ['--maxkb=1024', '--enforce-all']
+        # TODO: remove the following after resolved
+        exclude: |
+            (?x)^(
+                source/lmp/tests/lrmodel.pbtxt|
```

Comment:
Remove this line if #2561 is merged first. Otherwise, remove this line in #2561.